modules/mockk/src/commonMain/kotlin/io/mockk/impl/verify/UnorderedCallVerifier.kt

```diff
@@ -162,7 +163,7 @@ open class UnorderedCallVerifier(
         }
 
         captureBlocks.add {
-            for (call in allCallsForMockMethod) {
+            for (call in matchedCalls) {
```

Comment:
`captureAnswer` was called for non-matched calls, which results in incorrect capturing results